src/pages/PageNotFoundView.jsx

```diff
@@ -9,13 +12,20 @@ export default function PageNotFoundView() {
    ************************************/
 
   return (
-    <div className="page-not-found-view-container">
-      <div className="page-not-found-view-body">
-        <h1>404</h1>
-        <code>{location.pathname}</code>
-        <h3>Page Not Found</h3>
-        <p>Sorry, the page you are looking for doesn't seem to exist!</p>
-        <Link to="/">Go to Home</Link>
+    <div className='page-not-found-view-container'>
+      <div className='page-not-found-view-body'>
+        <div className='page-not-found-view-image-container'>
+          <NotFoundLogo />
+        </div>
+        <div className='page-not-found-view-text-container'>
+          <p>
+            404 Page not found, <strong>{location.pathname}</strong>
+          </p>
+          <p>Let's hurry; there is nothing to fear here</p>
```

Comment:
nit picky but there should be a period at the end of this

Reply:
You are being nit picky! All good, you're right.